src/components/application_manager/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -1202,7 +1211,9 @@ bool RegisterAppInterfaceRequest::IsApplicationWithSameAppIdRegistered() {
   ApplicationSetConstIt it = applications.begin();
   ApplicationSetConstIt it_end = applications.end();
   for (; it != it_end; ++it) {
-    if (mobile_app_id.CompareIgnoreCase((*it)->policy_app_id().c_str())) {
```

Comment:
@tkireva please Add additional function to CustonString - "Compare" (case sensitive one)